packages/cli/package.json

```diff
@@ -4,21 +4,23 @@
     "description": "Nango's CLI tool.",
     "type": "module",
     "main": "dist/index.js",
-    "bin": "./dist/index.js",
+    "bin": {
+        "nango": "dist/index.js"
+    },
     "types": "dist/nango-sync.d.ts",
     "keywords": [],
     "author": "bastien@nango.dev",
     "repository": {
         "type": "git",
-        "url": "https://github.com/NangoHQ/nango",
+        "url": "git+https://github.com/NangoHQ/nango.git",
         "directory": "packages/cli"
```

Comment:
unrelated to this PR. I just ran `npm pkg fix` to remove the warning when publishing